namespaces/default/pinnwand/README.md

```diff
@@ -0,0 +1,8 @@
+# pinnwand
+These manifests provision an instance of the pinnwand service used on https://paste.pythondiscord.com.
+
+A init-service is used to download the Python Discord banner logo and save it to a volume, as pinnwand expects it to be present within the image.
+
+## Secrets & config
+This deployment expects an env var named `PINNWAND_DATABASE_URI` to exist in a secret called `pinnwand-postgres-connection`.
+All other configuration can eb found in `defaults-configmap.yaml`.
```

Comment:
Typo
```suggestion
All other configuration can be found in `defaults-configmap.yaml`.
```

Reply:
Fixed in the force push, along with some other changes.

https://github.com/python-discord/kubernetes/compare/c0d746e373628e0f4976ff0d9fd0f9a06ed5109b..318a8e17ed7cb35a536bdc4bb5a96d141bd9f9bd